aspnetcore/tutorials/nano-server.md

```diff
@@ -160,12 +160,11 @@ New-IISConfigCollectionElement $modules -ConfigAttribute @{"name"="AspNetCoreMod
 
 ## Installing .NET Core Framework
 
-If you published a Framework-dependent (portable) app, .NET Core must be installed on the target machine. Execute the following PowerShell script in a remote PowerShell session to install the .NET Framework on your Nano Server.
+If your app is a [framework-dependent deployment (FDD)](/dotnet/core/deploying/#framework-dependent-deployments-fdd), .NET Core must be installed on the server. Use the [dotnet-install.ps1 PowerShell script](https://dot.net/v1/dotnet-install.ps1) in a remote PowerShell session to install .NET Core on your Nano Server. Pass the CLI version with the `-Version` switch:
```

Comment:
This 1st sentence should probably read something like "If your app is published using a Framework-Dependent Deployment (FDD)...". Thoughts?

Reply:
yeah ... but maybe with "as" ... since apps are published *as* an FDD or SCD. Stand-by ... I'll patch that right up in a sec.